wgpu/src/lib.rs

```diff
@@ -3448,9 +3448,7 @@ impl Drop for SurfaceTexture {
 impl Surface {
     /// Returns a vec of supported texture formats to use for the [`Surface`] with this adapter.
     /// Note: The first format in the vector is preferred
-    ///
-    /// Returns None if the surface is incompatible with the adapter.
```

Comment:
Let's leave this note in, but refactored to represent the new situation:

```
/// Returns an empty vector if the surface is incompatible with the adapter.
```

Reply:
Sure